models/tracker/src/migration.ts

```diff
@@ -148,6 +148,42 @@ async function upgradeIssueStatuses (tx: TxOperations): Promise<void> {
   }
 }
 
+async function upgradeIssueProjects (tx: TxOperations): Promise<void> {
```

Comment:
Please use  MigrationClient instead, to prevent not required transactions to appear.

Reply:
Done.